src/frontend/Komponenter/Behandling/VedtakOgBeregning/Skolepenger/SaksinformasjonSkolepenger.tsx

```diff
@@ -132,17 +142,24 @@ const SaksbehanldingsinformasjonUtdanning: React.FC<{ vilkår: IVilkår }> = ({
 };
 
 const SaksbehanldingsinformasjonUtgifter: React.FC<{ vilkår: IVilkår }> = ({ vilkår }) => {
-    const begrunnelseDokumentasjonUtdanning = vilkår.vurderinger.find(
-        (vurdering) => vurdering.vilkårType === AktivitetsvilkårType.DOKUMENTASJON_AV_UTDANNING
-    )?.delvilkårsvurderinger[1]?.vurderinger[0]?.begrunnelse;
+    const begrunnelseDokumentasjonUtgifterUtdanning = vilkår.vurderinger
+        .find(
+            (vurdering) => vurdering.vilkårType === AktivitetsvilkårType.DOKUMENTASJON_AV_UTDANNING
+        )
+        ?.delvilkårsvurderinger.flatMap((delvilkårsvurdering) => delvilkårsvurdering.vurderinger)
+        ?.find(
+            (vurdering) =>
+                vurdering.regelId ===
+                RegelIdDDokumentasjonUtdanning.DOKUMENTASJON_AV_UTGIFTER_UTDANNING
+        )?.begrunnelse;
 
     return (
         <div style={{ order: 1 }}>
             <Heading spacing size="small">
                 Saksbehandlers vurdering - Utgifter
             </Heading>
-            {begrunnelseDokumentasjonUtdanning ? (
-                <BreakWordBody>{begrunnelseDokumentasjonUtdanning}</BreakWordBody>
+            {begrunnelseDokumentasjonUtgifterUtdanning ? (
+                <BreakWordBody>{begrunnelseDokumentasjonUtgifterUtdanning}</BreakWordBody>
```

Comment:
Disse to komponentene er "identiske" bortsett fra to ting: 
`Header-tekst` og `RegelIdDokumentasjonUtdanning` 
Enten:
- Trekk ut den crazy "begrunnelseForRegelId" som egen funksjon eller gjør komponenten gjenbrukbar? Tror jeg heller mot det første for å lettere se hva som brukes hvor :)

Reply:
Tenker det er greit å ha to forskjellige komponenter, så trekker bare ut funksjonen 👍